README.md

```diff
@@ -19,6 +19,8 @@ Then you will create an instance of imported class, by choosing one of the follo
     ```
     var factory = CommonFactory.createFromArguments(args);
     ```
+    If you use arguments `--namespace` and `--boxName` here, `createFromKubernetes(namespace, boxName)` will be invoked.
+    If you use also `--contextName` in addition to `--namespace` and `--boxName`, `createFromKubernetes(namespace, boxName, contextName)` will be invoked
```

Comment:
Available set of arguments
Run with custom configs
--config .....
Run with Kube config (please more details in createFromKubernetes ...)
....